BUG: esa.hubble receives 301 error

I've just spotted these in the contests

```
E           requests.exceptions.HTTPError: Error 301:
E           <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
E           <html><head>
E           <title>301 Moved Permanently</title>
E           </head><body>
E           <h1>Moved Permanently</h1>
E           <p>The document has moved <a href="https://hst.esac.esa.int/tap-server/tap//sync">here</a>.</p>
E           </body></html>

astroquery/utils/tap/core.py:334: HTTPError
```

an example for a full log:

https://github.com/astropy/astroquery/actions/runs/3358764084/jobs/5566116707


cc @jespinosaar

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -26,6 +26,9 @@ esa.hubble
 - Refactored query_criteria to use ehst.archive table therefore making the query
   a lot faster. [#2524]
 
+- Update TAP url to avoid 301 HTTPError. [#2567]
+
+
 alma
 ^^^^
 

--- a/astroquery/esa/hubble/core.py
+++ b/astroquery/esa/hubble/core.py
@@ -47,8 +47,8 @@ class ESAHubbleClass(BaseQuery):
         super().__init__()
 
         if tap_handler is None:
-            self._tap = TapPlus(url="http://hst.esac.esa.int"
-                                    "/tap-server/tap/")
+            self._tap = TapPlus(url="https://hst.esac.esa.int"
+                                    "/tap-server/tap")
         else:
             self._tap = tap_handler
 

